docs/components/rotation.md

```diff
@@ -25,11 +25,12 @@ A-Frame uses a right-handed coordinate system. When aligning our right hand's
 thumb with a positive axis, our hand will curl in the positive direction of
 rotation.
 
-| Axis | Description                       | Default Value
-|------|-----------------------------------|---------------|
-| x    | Pitch, rotation about the X-axis. | 0             |
-| y    | Yaw, rotation about the Y-axis.   | 0             |
-| z    | Roll, rotation about the Z-axis.  | 0             |
+| Axis  | Description                           | Default Value
+|-------|---------------------------------------|---------------|
+| x     | Pitch, rotation about the X-axis.     | 0             |
+| y     | Yaw, rotation about the Y-axis.       | 0             |
+| z     | Roll, rotation about the Z-axis.      | 0             |
+| order | Order that the rotations are applied. | 'XYZ'         |
```

Comment:
XYZ order is the default in threejs (object3D.rotation is an Euler instance and the default is defined in https://github.com/mrdoob/three.js/blob/6e897f9a42d615403dfa812b45663149f2d2db3e/src/math/Euler.js#L321), but the default order if you use the rotation component is actually YXZ set here https://github.com/aframevr/aframe/blob/837023cca1a04b408b4acde4c09823e09dc58e6d/src/components/rotation.js#L14
The order is not configrable via the rotation component, so you can't add it to the table in the documentation like that.
You can change the order by setting `this.el.object3D.rotation.order = "XYZ"` though and setting directly in radians on `this.el.object3D.rotation.set(Math.PI/2, 0, 0)` for example, but be careful to not use again `this.el.setAttribute('rotation', "90 0 0")` because it will change again the order to YXZ.